### PR TITLE
docs: add angular-eslint and remove codelyzer from tooling

### LINKED ISSUE #41029: Codelyzer versus angular-eslint on resources page
# 📚 Docs or angular.io bug report

### Description

The [resources page](https://angular.io/resources?category=development) lists codelyzer, but not angular-eslint. Since tslint and in consequence codelyzer is deprecated and codelyzer is [being removed from the styleguide](https://github.com/angular/angular/pull/40992), we should consider

1. adding angular-eslint (should be a no-brainer)
2. removing codelyzer (or maybe still keep it?)

### What's the affected URL?**

https://angular.io/resources?category=development

### PR DESCRIPTION
**Pull request by ksivamuthu**
fixes #41029

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #41029

## What is the new behavior?

Add Angular ESLint in tooling and remove Codelyzer as described the issue #41029 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by ksivamuthu** [comment]
@googlebot I signed it!

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -230,11 +230,11 @@
...
           },
-          "d1": {
-            "desc": "Static analysis for Angular projects.",
+          "angular-eslint": {
+            "desc": "The tooling which enables ESLint to lint Angular projects.",
             "logo": "",
-            "title": "Codelyzer",
-            "url": "https://github.com/mgechev/codelyzer"
+            "title": "Angular ESLint",
+            "url": "https://github.com/angular-eslint/angular-eslint"
```
Comment (gkalpak) [inline aio/content/marketing/resources.json:237]: Suggestion:
Personally, I think it is better to have the browser automatically scroll to the README (by appending `#readme` to the URL), so people can read about the project and how to get started:

```suggestion
            "url": "https://github.com/angular-eslint/angular-eslint#readme"
```

WDYT?

**Review by gkalpak — APPROVED** [review]
Thx for taking this on, @ksivamuthu :+1:
I left on optional suggestion - otherwise lgtm 🚀

**Review by gkalpak — APPROVED** [review]
Reviewed-for: global-docs-approvers

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (ksivamuthu) [reply aio/content/marketing/resources.json]: Thanks. That's a good suggestion. I committed the changes.

**Pull request closed without merging**